app/frontend/courses/review/components/CoursesReview__Editor.res

```diff
@@ -1519,6 +1523,16 @@ let make = (
               </div>
             </div>
           }}
+          <div className="fixed bottom-0 inset-x-0 z-10">
+            <div>
+              {reviewNextButton(
+                state.nextSubmission,
+                filter,
+                state.nextSubmissionId,
+                "next-submission-button flex w-full items-center justify-center text-sm font-semibold bg-white border-t border-gray-200 px-5 py-4 hover:bg-primary-50 hover:text-primary-500 focus:ring-2 focus:ring-focusColor-500 ring-inset ",
```

Comment:
That button is used only once; Why pass the classes as a prop; why not add it directly to the element 🤔